lms/envs/common.py

```diff
@@ -1426,13 +1426,17 @@ def _make_locale_paths(settings):
     sorted(rooted_glob(PROJECT_ROOT / 'static', 'js/dashboard/**/*.js'))
 )
 discussion_js = (
-    rooted_glob(COMMON_ROOT / 'static', 'common/js/discussion/mathjax_include.js') +
     rooted_glob(PROJECT_ROOT / 'static', 'js/customwmd.js') +
     rooted_glob(PROJECT_ROOT / 'static', 'js/mathjax_accessible.js') +
     rooted_glob(PROJECT_ROOT / 'static', 'js/mathjax_delay_renderer.js') +
     sorted(rooted_glob(COMMON_ROOT / 'static', 'common/js/discussion/**/*.js'))
 )
 
+if 'common/js/discussion/mathjax_include.js' not in discussion_js:
+    discussion_js += (
```

Comment:
again, `discussion_js` init at lineno#1428 so I am not sure how this `if` condition be ever True?